PROFILING.md

```diff
@@ -0,0 +1,100 @@
+# Profiling Envoy
+
+## Install pprof
+
+From https://github.com/google/pprof#building-pprof
+
+```bash
+go get -u github.com/google/pprof
+```
+
+## Nighthawk's scripted benchmark
+
+Nighthawk comes with a [small framework and experimental benchmark suite](/benchmarks/) that
+will write `.prof` files to `/tmp/`:
+
+```bash
+bazel test --cache_test_results=no --compilation_mode=opt --cxxopt=-g --cxxopt=-ggdb3 //benchmarks:*
+```
+
+Note that it is possible to override Nighthawk's Envoy dependency
+to link against a local version, by adding a line to `.bazelrc`:
```

Comment:
It's cleaner and more minimally correct just to recommend adding this as a CLI option. Any CLI option can be put in `.bazelrc` at a user's discretion.